tests/sp2n/Test_2as_base.cc

```diff
@@ -1,13 +1,12 @@
 #include <Grid/Grid.h>
 
-#define verbose 0
+#define verbose 1
 
 using namespace Grid;
 
-int main(int argc, char** argv) {
-  Grid_init(&argc, &argv);
+static void antisymm_base() {
 
-  const int this_nc = 4;
+  const int this_nc = 6;
```

Comment:
If we already have this parameter, should we actually check for multiple values of nc, e.g. in a for loop?

Reply:
Yes, I just didn't know how to loop over template parameters

Reply:
I can try and set something up. Ain't gonna be super pretty but at least will have significantly improved coverage.

Reply:
I mean the very simple thing would be
```cpp
template<int this_nc>
static void antisymm_base() { ... }

int main() {
...
antisymm_base<2>();
antisymm_base<4>();
...
}
```
Not quite a loop yet, but the easiest by far. We can see if it works fine like that and then maybe have a loop in the main function if necessary.